app/src/main/res/menu/pullrequest_menu.xml

```diff
@@ -5,6 +5,11 @@
         android:id="@+id/pull_merge"
         android:title="@string/pull_request_merge"
         app:showAsAction="ifRoom|withText" />
+    <item
+        android:id="@+id/pull_review"
+        android:title="Review Changes"
```

Comment:
Extract the string to `strings.xml`.

Code style: There should be 1 blank line above and below this item.